CHANGELOG.md

```diff
@@ -9,9 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 ### Added
   - `disabledRules` extension property to disable rules usage by id (#267)
 ### Changed
-  - Update Gradle to `5.6` version
+  - Update Gradle to `5.6.2` version
   - Update Kotlin to `1.3.50` version
   - Set default ktlint version to `0.34.2`
+  - Update Android Gradle plugin to `3.5.0` version
+  - Minimal supported Gradle version updated to `4.5.1`
```

Comment:
`LOWEST_SUPPORTED_GRADLE_VERSION != 4.5.1`

Reply:
:man_facepalming:

Reply:
@JLLeitschuh fixed in https://github.com/JLLeitschuh/ktlint-gradle/pull/282/commits/62ac3b0761e2f07d19dcc685747aac100b313506